.github/ISSUE_TEMPLATE/new-release.md

```diff
@@ -26,16 +26,33 @@ future releases.
   - [ ] Push the tag with `git push $VERSION`.
   - [ ] Check that new container images are published for the tag.
     - ```
-      for i in nri-resource-policy-topology-aware nri-resource-policy-balloons nri-resource-policy-template nri-config-manager nri-memory-qos nri-memtierd; do \
-          skopeo inspect --format "$i: {{.Digest}}" docker://ghcr.io/containers/nri-plugins/$i:$VERSION;
+      for i in nri-resource-policy-topology-aware nri-resource-policy-balloons nri-resource-policy-template nri-config-manager nri-memory-qos nri-memtierd nri-sgx-epc; do
+          skopeo inspect --format "$i: {{.Digest}}" docker://ghcr.io/containers/nri-plugins/$i:$VERSION
       done
+      # Notes:
+      # You can also do an image artifact verification with this repo script:
+      ./scripts/release/check-artifacts.sh --images $VERSION
       ```
   - [ ] Finalize the new *draft* release created by CI
     - [ ] Check that all artefacts (Helm charts) were uploaded
     - [ ] Write the change log to the release.
     - [ ] Get the change log OK'd by other maintainers.
   - [ ] Publish the draft as a release.
-  - [ ] Verify that the Helm repo was updated
+  - [ ] Verify that all the Helm repo was updated
+    - ```
+      wget https://raw.githubusercontent.com/containers/nri-plugins/refs/heads/gh-pages/index.yaml
```

Comment:
nit: should we check the actual helm repo we're advertising, i.e. https://containers.github.io/nri-plugins/index.yaml ?

Reply:
Thanks for spotting that... :see_no_evil: